pagopa/io-app
# fix: [IOBP-793] Unauthorized status mapping from checkout APIs

## Short description
This PR handles the missing 401 status from all checkout APIs

## List of changes proposed in this pull request
- Updated the API checkout and payments wallet specs with the status 401 mapped
- Added the check to not go in failure when the status response is `401` since it will be handled by the `withPaymentsSessionToken` decorator that handles the unauthorized request.

## How to test
To test this PR you have to switch on the UAT env. (pagoPA Test feature flag in the profile section) and then start a new payment flow using the following info:
- Notice number: 302043344343434343
- CF: 77777777777
Then before you put the CF and go ahead by pressing the "Continue" button, wait until the session expires.
